homeassistant/components/fritz/translations/de.json

```
@@ -0,0 +1,43 @@
+{
+  "config": {
+    "flow_title": "FRITZ!Box Tools: {name}",
+    "step": {
+      "confirm": {
+        "title": "Setup FRITZ!Box Tools",
+        "description": "Discovered FRITZ!Box: {name}\n\nSetup FRITZ!Box Tools to control your {name}\nDocumentation: https://github.com/mammuth/ha-fritzbox-tools",
```

Urls should be excluded from translations.

```suggestion
        "description": "Discovered FRITZ!Box: {name}\n\nSetup FRITZ!Box Tools to control your {name}\nDocumentation: {docs_url}",
```

Then be sure to include it in `config_flow.py`